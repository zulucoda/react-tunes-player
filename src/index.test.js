import React from 'react';
import { render } from '@testing-library/react';
import ReactTunesPlayer from '.';

describe('React Tunes Player - Render Tests', () => {
  test('should render new player by default', () => {
    const { container } = render(<ReactTunesPlayer tunes={[]} />);
    expect(container).toMatchSnapshot();
  });

  test('should render old player by setting oldPlayer prop', () => {
    const { container } = render(
      <ReactTunesPlayer tunes={[]} oldPlayer={true} />,
    );
    expect(container).toMatchSnapshot();
  });
});
