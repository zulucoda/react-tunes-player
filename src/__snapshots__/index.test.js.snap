// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`React Tunes Player - Render Tests should render new player by default 1`] = `
<div>
  <div
    class="sc-jdAMXn eBAAmP"
  >
    <div
      class="sc-bjUoiL gvnUrq"
    >
      <svg
        viewBox="0 0 512 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M32 464h448L256 48 32 464zm248-64h-48v-48h48v48zm0-80h-48v-96h48v96z"
        />
      </svg>
      <span
        class="sc-dIouRR evIpjP"
      >
        Warning! No tunes loaded in player.
      </span>
    </div>
  </div>
</div>
`;

exports[`React Tunes Player - Render Tests should render old player by setting oldPlayer prop 1`] = `
<div>
  <div
    class="sc-bczRLJ gDWqKd"
  >
    <div
      class="sc-gsnTZi ljuRqf"
    >
      <div
        class="sc-bjUoiL gvnUrq"
      >
        <svg
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M32 464h448L256 48 32 464zm248-64h-48v-48h48v48zm0-80h-48v-96h48v96z"
          />
        </svg>
        <span
          class="sc-dIouRR evIpjP"
        >
          Warning! No tunes loaded in player.
        </span>
      </div>
    </div>
  </div>
</div>
`;
